### PR TITLE
Extending readme

### PR DESCRIPTION
**Pull request by lawrab**
Minor changes to the readme file to provide context and an overview of what Slipstream is and can do for a user.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,63 @@
...
+
+say_name = {}
+say_name["sassy_mcsassypants"] = "Sassy"
+say_name["larryrabbets"] = "Larry"
+
+function string.starts(String,Start)
+   return string.sub(String,1,string.len(Start))==Start
+end
+
+function handle(event)
```
Comment (dennis) [inline README.md:47]: Seems like there is some indentation problems: 
image

File: README.md [hunk 2]
```diff
@@ -1,17 +1,63 @@
...
+    print(event_to_json(event))
```
Comment (dennis) [inline README.md:48]: Not really needed for this example

File: README.md [hunk 3]
```diff
@@ -1,17 +1,63 @@
...
+    if(event.EventType == "TwitchReceivedMessage") then
+		if string.starts(event.Message, "!say ") then
+			if whitelist[string.lower(event.From)] then
+                local text = string.lower(string.sub(event.Message, 6))
+				audio:say(say_name[event.From:lower()] .. " said ... " .. text, 1)
```
Comment (dennis) [inline README.md:53]: Just changed that syntax, so you probably need "AudioPlugin" as the first argument

File: README.md [hunk 4]
```diff
@@ -1,17 +1,63 @@
...
+- Interacting with IoT devices
+- Playing audio samples
+- Text to speech
+
+Example script
+```Lua
+local whitelist = {}
+
+whitelist["sassy_mcsassypants"] = true
+whitelist["larryrabbets"] = true
```
Comment (dennis) [inline README.md:37]: Instead of setting these to true, you could use the name you want to use and then you can remove `say_name`

**Review by dennis — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (lawrab) [reply README.md]: I will replace this when I have a working script

File: README.md (hunk 4 above)
Comment (lawrab) [reply README.md:37]: yup, was just a quick hack while Sassy was testing, we probably need better examples in the readme and then examples in the scripts folder

File: README.md (hunk 2 above)
Comment (lawrab) [reply README.md]: Will come up with a better example

File: README.md (hunk 1 above)
Comment (lawrab) [reply README.md:47]: untabified the file, there were a mix of tabs and spaces used in the readme

**Review by dennis — APPROVED** [review]

**Pull request merged**